platform-sdk/platform-apps/tests/AddressBookTestingTool/AddressBookTestingToolInstructions.md

```diff
@@ -0,0 +1,175 @@
+# AddressBookTestingTool Instructions
```

Comment:
I've changed it to README.md

I agree this is a stop-gap for not yet having automated testing capability.    That will hopefully be added soon. 

I created the volume of detail because the behavior is being handed off to another team and I wanted a walk-through of the complexity of behavior.